contracts/neutron_interchain_txs/src/contract.rs

```diff
@@ -246,6 +252,7 @@ fn execute_undelegate(
         interchain_account_id.clone(),
         vec![any_msg],
         "".to_string(),
+        DEFAULT_TIMEOUT_SECONDS,
```

Comment:
y no option to set timeout for undelegation calls?

Reply:
I didn't need this in integration tests, but it makes sense to have this functionality anyway. Added it.